### ISSUE TITLE
OnCall Helm Installation Verification Not Working

### ISSUE THREAD

**Issue by lusid** [grafana/oncall#951]
I have a separate Grafana installation that I'm trying to connect to the OnCall install that I did with Helm. This was working with helm chart v1.0.6 and oncall image v1.0.42 when I had to manually shell into the container and run a command to get the token, but now that it is trying to automatically install, it no longer works.

grafana v9.1.5
oncall helm chart v1.0.12
oncall docker v1.1.5

These are the helm values I'm using:

```
        fullnameOverride: 'oncall'
        grafana:
          enabled: false
        cert-manager:
          enabled: false
        ingress-nginx:
          enabled: false
        ingress:
          enabled: false
        redis:
          clusterDomain: management.internal
        mariadb:
          clusterDomain: management.internal
        rabbitmq:
          clusterDomain: management.internal
```

In the Grafana plugin, I tell it to connect to http://oncall-engine-headless.monitoring.svc.management.internal:8080, which is the headless service I configured manually (because I need it to be headless for my Grafana installation to be able to reach it). It errors with the following message:

```
An unknown error occured when trying to install the plugin. Are you sure that your OnCall API URL, http://oncall-engine-headless.monitoring.svc.management.internal:8080, is correct?
Refresh your page and try again, or try removing your plugin configuration and reconfiguring.
```

When I try to connect to the same endpoint that Grafana is trying to use (http://oncall-engine-headless.monitoring.svc.management.internal:8080/api/internal/v1/plugin/self-hosted/install) I get the following message:

```
{"detail":"You do not have permission to perform this action."}
```

Here are the logs from oncall-engine:

```
2022-12-05 19:25:14 source=engine:app google_trace_id=none logger=root inbound latency=0.028375 status=403 method=POST path=/api/internal/v1/plugin/self-hosted/install content-length=0 slow=0 
2022-12-05 19:25:14 source=engine:app google_trace_id=none logger=django.request Forbidden: /api/internal/v1/plugin/self-hosted/install
2022-12-05 19:25:14 source=engine:uwsgi status=403 method=POST path=/api/internal/v1/plugin/self-hosted/install latency=0.029817 google_trace_id=- protocol=HTTP/1.1 resp_size=263 req_body_size=0
```

Any idea why it would be complaining about permissions and how to fix it?

**Comment by timchenko-a** [comment]
Facing the same issue with the next setup:
Grafana (separate installation): v8.5.13
Helm chart version: 1.0.11
Grafana oncall engine version: v1.1.0

**Comment by timchenko-a** [comment]
Just in case, I was able to fix this by explicitly setup:
```
    env:
      - name: GRAFANA_API_URL
        value: grafana url
```
Most probably this will be fixed permanently with the next helm chart version release:
https://github.com/grafana/oncall/blob/dev/helm/oncall/values.yaml#L296-L297

**Comment by lusid** [comment]
Unfortunately, even though this seems like exactly what I needed, this still doesn't fix the problem for me. 🤷‍♂️

**Comment by lusid** [comment]
This solved it for me with the latest v1.1.0 helm chart release. Thanks!
